test/builder-haddock/default.nix

```diff
@@ -82,6 +77,8 @@ in
       touch $out
     '';
 
-    meta.platforms = platforms.all;
-    meta.disabled = stdenv.hostPlatform.isMusl;
+    meta = {
+      platforms = platforms.all;
+      disabled = stdenv.hostPlatform != stdenv.buildPlatform || stdenv.hostPlatform.isMusl;
```

Comment:
I previously had a horrible time doing these sort of platform checks in order to remove jobs from the Hydra jobset.

What I think was happening, is that Hydra was getting the job list by evaluating with its native linux platform. So I was unable to remove jobs for e.g. macOS only.

But I was probably doing something wrong - is this working alright on Hydra?

Reply:
I think it was working before! The platform it's evaluating on shouldn't matter (unless you use `buildins.currentSystem` - this is checking the host/build platforms from stdenv, so should be okay.